plugins/processors/awsappsignals/config/config_test.go

```diff
@@ -0,0 +1,34 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+package config
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestValidatePassed(t *testing.T) {
+	config := Config{
+		Resolvers: []Resolver{NewEKSResolver("test"), NewGenericResolver()},
```

Comment:
Checks failing due to this being empty. Please pass it an empty string

Reply:
Updated.